foundations/Cargo.toml

```diff
@@ -77,6 +77,12 @@ telemetry = [
     "telemetry-server",
 ]
 
+tokio-metrics = [
+    "tokio/rt",
+    "metrics",
+    "serde/rc",
+]
```

Comment:
Should it be in any of the default feature sets? If not, we need to explain why not in docs

Reply:
I don't think we should put it in there until we stabilize this. When we do, we can revisit this.